pkg/aqua/aqua.go

```diff
@@ -0,0 +1,237 @@
+package aqua
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/Benchkram/bob/pkg/packagemanager"
+	"github.com/Benchkram/errz"
+	"github.com/aquaproj/aqua/pkg/controller"
+	"gopkg.in/yaml.v3"
+)
+
+// Constants used internally
+const (
+	AQUA_ROOT      = ".bob/.aqua"
+	AQUA_VERSION   = "v0.13.0"
```

Comment:
For the package search functionality it has to somehow call the registry. Should we allow a latest flag which pings github release page after 24h passed?  Will save us from regular releases.